base/process.jl

```diff
@@ -335,6 +335,20 @@ end
 pipe_reader(p::Process) = p.out
 pipe_writer(p::Process) = p.in
 
+"""
+    getpid(p::Process) -> Int32
```

Comment:
Why not return `Int`?

Reply:
Right. I just copied that line from the `Base.getpid()` doc.  The actual type is `Cint`, but I think that is the same as `Int32` on all the supported platforms.

Reply:
Yes, this was the right choice (matching `getpid()`)

Reply:
What about changing that to `Int` for both methods in another PR? for example, one method of `kill` (with `ClusterManager`) accepts a pid as `Int`.

Reply:
@rfourquet - isn't that a different ID?  I thought `kill(::ClusterManager, ..)` takes the worker id, not the OS process id.

Reply:
It seems you are right.